README.md

```diff
@@ -81,6 +81,7 @@ Option | Type | Default | Description
 `position` | `String` | `topleft` | One of the valid Leaflet [control positions](http://leafletjs.com/reference.html#control-positions).
 `zoomToResult` | `Boolean` | `true` | If `true` the map will zoom the result after geocoding is complete.
 `useMapBounds` | `Boolean` or <br> `Integer` | `12` | Determines if and when the geocoder should begin using the bounds of the map to enchance search results. If `true` the geocoder will always return results in the current map bounds. If `false` it will always search the world. If an integer like `11` is passed in the geocoder will use the bounds of the map for searching if the map is at a zoom level equal to or greater than the integer. This mean the geocoder will prefer local results when the map is zoomed in.
+`searchBounds` | `L.latLngBounds` | `false` | This will supercede useMapBounds if set to anything but true.  If set, the geocoder will use the provided bounds to make the suggestion search despite the zoom level and map bounds.
```

Comment:
seems simpler to me for this new property to be `L.latLngBounds` instead of `boolean || L.latLngBounds`.

then the wording in the doc could be:
> If set, the geocoder will filter results using the provided static bounding box regardless of the current zoom level of the map (ie: supercedes `useMapBounds`).